radio_handler.c

```diff
@@ -0,0 +1,63 @@
+#include "radio_handler.h"
+#include "serialize.h"
+#include <string.h> // for memcpy
+
+static enum VALVE_STATE inj_valve_state = VALVE_UNK;
+static enum VALVE_STATE vent_valve_state = VALVE_OPEN;
+
+enum VALVE_STATE radio_get_expected_inj_valve_state(void)
+{
+    return inj_valve_state;
+}
+
+enum VALVE_STATE radio_get_expected_vent_valve_state(void)
+{
+    return vent_valve_state;
+}
+
+void radio_handle_input_character(uint8_t c)
+{
+    static char message[STATE_COMMAND_LEN] = {0};
+    static uint8_t chars_received = 0;
+
+    if(c == '{') {
```

Comment:
`{` feels like a magic character, can you define it as STATE_COMMAND_HEADER or something?

Reply:
Fixed in ae3c242.